src/app/space/app-launcher/app-launcher.module.ts

```diff
@@ -1,23 +1,18 @@
 import { CommonModule } from '@angular/common';
-import { HttpClient } from '@angular/common/http';
 import { NgModule } from '@angular/core';
 import {
   AuthHelperService,
   Config,
-  DependencyEditorTokenProvider,
   HelperService,
-  TokenProvider,
-  URLProvider
+  TokenProvider
```

Comment:
Why do we need the `TokenProvider`? It is just wrapping one method from `AuthenticationService` to get token. This module already provides `AuthenticationService`, so why not use it directly?

Reply:
It's used to abstract away how the token is obtained. On the standalone launcher a different mechanism is used.